post.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="post.css">
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+
+    <title>Nordic Rose</title>
+</head>
+
+<body>
+    <header class="header">
+
+        <div class="top_padding_nordic">
+            <h1>
+                <picture>
+                    <source media="(max-width:854px)" srcset="photos\mobile_rose.png" />
+                    <source media="(min-width:854px)" width="395" height="38" srcset="photos\desktop_rose.png" />
+                    <img class="rose" width="217" height="24" src="photos\mobile_rose.png" alt="NORDIC ROSE">
+                </picture>
+            </h1>
+        </div>
+        <div class="desktop_menu">
+            <a href="" class="first_part">BLOG</a>
+            <a href="" class="second_part">ABOUT</a>
+            <a href="" class="third_part">LINKS</a>
+            <a href="" class="fouth_part">PROJECTS</a>
+        </div>
+        <div class="top_padding_burger"><img width="25" height="16" src="photos\burger.png" alt="burger">
+        </div>
+    </header>
+    <artcile>
```

Comment:
1. incorrect element name
2. it does not have closing tag 
```suggestion
    <article>
```